src/tests.js

```diff
@@ -0,0 +1,25 @@
+const pkgOk = require('./pkg-ok.js');
+const esCheck = require('./es-check.js');
+const pkgCanInstall = require('./pkg-can-install.js');
+const canRequire = require('./can-require');
+
+const tests = {
```

Comment:
If we ever find that we need to add more tests in the future it will be easy now. Furthermore tests now have names that make sense, rather than being named after binaries.